pkg/ui/Makefile

```diff
@@ -27,8 +27,8 @@ all: lint test $(GOBINDATA_TARGET)
 protos:
 	$(MAKE) -C $(ORG_ROOT) -f cockroach/build/protobuf.mk
 
-# Running `go generate` will call this target. Update this if you add new
-# generated files.
+# Running `make generate` from the root will call this target via `go generate`.
```

Comment:
How is that inaccurate? `make generate` from the root will run `go generate`, which will exactly run this target https://github.com/cockroachdb/cockroach/blob/master/pkg/ui/ui.go#L21